cmd/format-coverage.go

```diff
@@ -38,26 +38,33 @@ func (f CoverageFormatter) Save() error {
 	}
 	err = in.Parse()
 	if err != nil {
-		return err
+		return errors.WithStack(err)
 	}
 
 	var out io.Writer
 	if formatOptions.Print || formatOptions.Output == "-" {
 		out = os.Stdout
 	} else {
-		os.MkdirAll(filepath.Dir(formatOptions.Output), 0755)
+		err = os.MkdirAll(filepath.Dir(formatOptions.Output), 0755)
+		if err != nil {
+			return errors.WithStack(err)
+		}
 		out, err = os.Create(formatOptions.Output)
 		if err != nil {
-			return err
+			return errors.WithStack(err)
 		}
 	}
 
 	rep, err := in.Format()
 	if err != nil {
-		return err
+		return errors.WithStack(err)
 	}
 
-	return rep.Save(out)
+	err = rep.Save(out)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+	return nil
```

Comment:
The changes in this file feel un-related to sum-coverage. If so, could we break out their own commit and/or a separate PR?